index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Belo Horizonte/MG - Brazil</title>
+    </head>
+    <body>
+        <header>
+            <h1 style="text-align: center">Welcome to <br> BELO HORIZONTE</h1>
+        </header>
+        <nav>
+            <ul>
+                <a href="#home"><li>Home</li></a>
+                <a href="#about"><li>About</li></a>
+                <a href="#history"><li>History</li></a>
+                <a href="#location"><li>Location</li></a>
+                <a href="#gastronomy"><li>Gastronomy</li></a>
+                <a href="#more"><li>More</li></a>
+                <a href="#contact"><li>Contact</li></a>
+            </ul>
```

Comment:
the li elements need to include the a href

```
<ul>
<li> list item  1</li>
<li> list item 2 </li>
</ul>
```

Reply:
got it